packages/mobile/src/components/remix-carousel/RemixCarousel.tsx

```diff
@@ -0,0 +1,28 @@
+import { useTracks } from '@audius/common/api'
+import { type ID, type Track } from '@audius/common/models'
+
+import type { CardListProps } from 'app/components/core'
+import { CardList } from 'app/components/core'
+
+import { RemixContestCard } from './RemixContestCard'
+
+type ListProps = Omit<CardListProps<Track>, 'data'>
+
+export type UserListProps = {
+  trackIds: ID[] | undefined
+  onCardPress?: (user_id: ID) => void
+} & Partial<ListProps>
+
+export const RemixCarousel = (props: UserListProps) => {
+  const { trackIds, ...other } = props
+  const { data: tracks } = useTracks(trackIds)
```

Comment:
are these tracks already fetched? if so you dont need this line. if so, can we still just pass ids into the card-list and do track-ids all the way down?